apps/dashboard/app/javascript/batch_connect_sessions.js

```diff
@@ -3,26 +3,37 @@
 import { bcIndexUrl, bcPollDelay } from './config';
 import { bindFullPageSpinnerEvent } from './utils';
 import { pollAndReplace } from './turbo_shim';
-import { ariaNotify } from './aria_live_notify';
+import { ariaNotify, pushNotify } from './notify';
 
 const sessionStatusMap = new Map();
 
 function checkStatusChanges() {
   const sessionCards = document.querySelectorAll('[data-bc-card]');
   
   sessionCards.forEach((card) => {
+    const sessionTitle = card.dataset.title;
     const sessionId = card.dataset.id;
-    const newStatus = card.dataset.status;
+    const jobId = card.dataset.jobId;
+    const currentStatus = card.dataset.status;
 
     if(sessionStatusMap.has(sessionId)) {
       const oldStatus = sessionStatusMap.get(sessionId);
-      if(oldStatus !== newStatus) {
-        sessionStatusMap.set(sessionId, newStatus);
-        const sessionTitle = card.dataset.title;
-        ariaNotify(`${sessionTitle} is now ${newStatus}.`);
+      if(oldStatus !== currentStatus) {
+        sessionStatusMap.set(sessionId, currentStatus);
+        ariaNotify(`${sessionTitle} is now ${currentStatus}.`);
+        pushNotify(`${sessionTitle} (${jobId}) is now ${currentStatus}.`, {
+          tag: `session-${sessionId}`,
+        });
       }
     } else {
-      sessionStatusMap.set(sessionId, newStatus);
+      sessionStatusMap.set(sessionId, currentStatus);
+    }
+
+    const minutesRemaining = parseInt(card.dataset.minutesRemaining);
+    if (minutesRemaining === 15 || minutesRemaining === 5) {
```

Comment:
Poll delays are set by the center. Let's say the delay is 6 minutes. This may never evaluate to true under certain conditions like that.

Seems like we need `>` instead of `===` with maybe some mechanism to keep track of what cards have notified because it could spam notifications under that limit.

Reply:
> something persistent across reloads 

Yea we likely need to use `localStorage`.